client/src/main/java/org/spine3/type/KnownTypes.java

```diff
@@ -245,8 +245,7 @@ public boolean apply(@Nullable TypeUrl input) {
      *
      * @throws UnknownTypeException if there is no Java class for the passed type URL
      */
-    @VisibleForTesting
-    public static <T extends Message> Class<T> getClass(TypeUrl typeUrl) {
+    static <T extends Message> Class<T> getJavaClass(TypeUrl typeUrl) throws UnknownTypeException {
```

Comment:
Why don't we move all this logic to `TypeUrl` letting it manage its form by itself?